### PR TITLE
dbeaver/pro#2137 auto relayout

### PR DESCRIPTION
**Pull request by serjiokov**
Added relayout diagram support for cases:
1. Modify visible attributes - view styles and show attributes
2. Switch between notaions and routing

### CODE REVIEW

**Comment by serjiokov** [comment]
image

File: plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorPart.java [hunk 1]
```diff
@@ -1215,6 +1216,7 @@ public void run()
                     }
                 }
             }
+            refreshDiagram(true, true);
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorPart.java]: As far as I remember, we intentionally don't refresh full diagram here as if the diagram is large it take noticeable amount of time. Isn't there another way of rebuilding relations without it?

**Comment by serge-rider** [comment]
Diagram refresh is too expensive. Use rearrangeDiagram instead.

**Comment by serjiokov** [comment]
Unfortunatly method use "getDiagramPart().rearrangeDiagram();" doesn't help
image

**Comment by E1izabeth** [comment]
Can you don't reload metadata than at least?

**Review by E1izabeth — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**